### DISCUSSION TITLE
Stopped working, Mac to Android

### DISCUSSION THREAD

**Discussion by mattsh2** [localsend/localsend#1182] [Discussion (English, International)]
Worked well for a while. Then 1 day I decided to change the download destination on the phone and it all stopped working. Uninstalled and reinstalled. Still not working. Message is... 

PathAccessException: Cannot open file, path = '/storage/emulated/0/Download/Filename.mp4' (OS Error: Operation not permitted, errno = 1)

**Comment by Tienisto** [comment]
Please try to change the file name
